src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -63,6 +72,74 @@ import {
 const ImagesTable = () => {
   const [page, setPage] = useState(1);
   const [perPage, setPerPage] = useState(10);
+  const [isTypeSelectOpen, setIsTypeSelectOpen] = useState(false);
+  const [isVersionSelectOpen, setIsVersionSelectOpen] = useState(false);
+  const [filters, setFilters] = useState<{
+    imageType: string[];
+    version: string[];
+  }>({
+    imageType: [],
+    version: [],
+  });
+  const experimentalFlag = process.env.EXPERIMENTAL;
```

Comment:
do we want to use feature flag here?

Reply:
I do want to but have been having lots of issues testing against stage the past week to verify that the flag actually works.